src/Microsoft.Windows.CsWin32/Generator.cs

```diff
@@ -3659,6 +3659,20 @@ private StructDeclarationSyntax DeclareStruct(TypeDefinitionHandle typeDefHandle
         // Add the additional members, taking care to not introduce redundant declarations.
         members.AddRange(additionalMembers.Where(c => c is not StructDeclarationSyntax cs || !members.OfType<StructDeclarationSyntax>().Any(m => m.Identifier.ValueText == cs.Identifier.ValueText)));
 
+        switch (name.Identifier.ValueText)
+        {
+            case "POINTF":
+            case "POINT":
+            case "RECT":
+            case "RECTF":
```

Comment:
You mention `RECTF` and `SIZEF` here but you don't have templates for them.

Reply:
I think SIZEF and RECTF are not metadata types. So I removed the templates.